### DISCUSSION TITLE
How to run the project locally and develop in the linux environment

### DISCUSSION THREAD

**Discussion by GracefulTing** [coder/code-server#3498] [Q&A]
I can run successfully locally by downloading the binary package in the linux environment, but I want to view the debugging results after modifying the code-server code locally. In this case, the binary file cannot be modified, so how to run it in this situation? According to the command, an error is reported, but it can't get through?

**Comment by code-asher** [comment]
For anyone interested in this we are discussing it here: https://github.com/cdr/code-server/discussions/3488#discussioncomment-795994
